docs/reference/config-files/workspace.md

```diff
@@ -31,6 +31,9 @@ tailpipe query --workspace development
 |---------------------|-------------|-----------------------------------------
 | `log_level`         | off         | Set the logging output level
 | `update_check`      | `true`      | Enable or disable automatic update checking.
+| `memory_max_mb`     | `unlimited`     | Caps CLI memory usage and determines worker count.
```

Comment:
I think the value is `0` which means unlimited.  maybe list it as: `0` (unlimited), eg:
```
| `memory_max_mb`             | `0` (unlimited) | Caps CLI memory usage and determines worker count.
| `plugin_max_memory_mb` | `0` (unlimited) | Sets soft memory cap per plugin.
| `max_temp_dir_mb`            | `0` (unlimited) | Limits JSONL temp file size on disk.
```